src/Emulator/Visuals/Visuals.tsx

```diff
@@ -71,25 +94,27 @@ const Visuals = () => {
     <div style={{
       height: CONTAINER_HEIGHT,
       width: CONTAINER_WIDTH,
+      marginTop: '2.5vh',
+      marginLeft: '1.5vw',
       overflow: 'hidden',
       position: 'relative',
+      borderRadius: '10px',
+      backgroundColor: '#212529',
+      boxShadow: '0 0 0 0.4vw #454950',
+      zIndex: -1,
     }}>
       <div className="icons">
-        <button className='iconButtons' onClick={toggleHideImage}>
-          <FontAwesomeIcon className='icon' icon={faEyeSlash} />
-        </button>
-        <button className='iconButtons' onClick={recenterImage}>
-          <FontAwesomeIcon className='icon' icon={faUndo} />
-        </button>
         {!hideImage &&
         <>
-        <button className='iconButtons' onClick={zoomIn} style={{ cursor: hideImage ? 'default' : 'pointer' }}>
-          <FontAwesomeIcon className='icon' icon={faSearchPlus} />
-        </button>
-        <button className='iconButtons' onClick={zoomOut} style={{ cursor: hideImage ? 'default' : 'pointer' }}>
-          <FontAwesomeIcon className='icon' icon={faSearchMinus} />
-        </button>
+        <HighlightButton hoverComponent={<GreenZoomInButton />} component={<ZoomInButton />} className='iconButtons' onClick={zoomIn} style={{ cursor: hideImage ? 'default' : 'pointer' }}>
+        </HighlightButton>
+        <HighlightButton hoverComponent={<GreenZoomOutButton />} component={<ZoomOutButton />} className='iconButtons' onClick={zoomOut} style={{ cursor: hideImage ? 'default' : 'pointer' }}>
+        </HighlightButton>
```

Comment:
This can all be self closing.
Also please break these out into multiple lines.
e.g.
```jsx
<HighlightButton
  hoverComponent={<GreenZoomInButton />}
  component={<ZoomInButton />}
  className='iconButtons'
  onClick={zoomIn}
  style={{ cursor: hideImage ? 'default' : 'pointer' }}
/>
```

It makes it easier to follow and see what the component is doing.